### PR TITLE
Use wp_print_inline_script_tag() to construct module script

### PR DESCRIPTION
**Pull request by westonruter**
## Summary

~_This is a sub-PR or #983. Review/merge it first._~

By using `wp_print_inline_script_tag()` we can ensure that the CSP `nonce` attribute can be injected, as well as any other desired attributes.

## Checklist

- [x] PR has either `[Focus]` or `Infrastructure` label.
- [x] PR has a `[Type]` label.
- [x] PR has a milestone or the `no milestone` label.

### CODE REVIEW

File: tests/modules/js-and-css/embed-optimizer/embed-optimizer-test.php [hunk 1]
```diff
@@ -131,4 +130,32 @@ public function get_data_to_test_filter_oembed_html_data(): array {
 			),
 		);
 	}
+
+	/**
+	 * Test printing the script lazy-loader.
+	 *
+	 * @covers ::embed_optimizer_lazy_load_scripts
+	 * @dataProvider get_data_to_test_filter_oembed_html_data
```
Comment (adamsilverstein) [inline tests/modules/js-and-css/embed-optimizer/embed-optimizer-test.php]: this dataProvider isn't used here, right?
```suggestion
```

**Review by adamsilverstein — COMMENTED** [review]
left one comment

File: tests/modules/js-and-css/embed-optimizer/embed-optimizer-test.php (hunk 1 above)
Comment (westonruter) [reply tests/modules/js-and-css/embed-optimizer/embed-optimizer-test.php]: Good catch

**Review by adamsilverstein — COMMENTED** [review]
Marvelous!

**Review by adamsilverstein — APPROVED** [review]

**Pull request merged**